HPCC-13951 Add suspended state to summary page

miguelvazq opened the pull request:
Give the user the visual cue and checkbox state (read-only) if suspended by anything other than user.

Signed-off by: Miguel Vazquez miguel.vazquez@lexisnexis.com

miguelvazq commented:
@GordonSmith Please review.

richardkchapman commented on esp/src/eclwatch/QuerySetDetailsWidget.js:
```diff
@@ -214,10 +214,10 @@ define([
                 this.workunitsTab.set("title", newValue);
             } else if (name === "Suspended") {
                 dom.byId(this.id + "SuspendImg").src = newValue ? dojoConfig.getImageURL("suspended.png") : "";
+            } else if (name === "Suspended") {
```
This is a dup of the prior condition. Was it supposed to say "SuspendBy" ?

richardkchapman replied:
Or SuspendedBy ?

miguelvazq replied:
Good catch but the API responds with "Suspended:true" if it is suspended by user therefore, I throw in a different icon for that. But, now that I look at this diff I see that Logical Files fix should not be in there I already committed that.

miguelvazq replied:
What I added is suspended by mixed node states or by cluster.

miguelvazq replied:
You are correct. Let me fix this.

Commit:
HPCC-13951 Add suspended state to summary page

Give the user the visual cue and checkbox state (read-only) if suspended by anything other than user.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

miguelvazq commented:
Fixing with https://github.com/hpcc-systems/HPCC-Platform/pull/7638
